jina/types/document/__init__.py

```diff
@@ -262,6 +264,11 @@ def __init__(
         if self._pb_body.id is None or not self._pb_body.id:
             self.id = random_identity(use_uuid1=True)
 
+        # check if there are mutually exclusive content fields
+        if _contains_conflicting_content(**kwargs):
+            raise ValueError(
+                f'Document content fields are mutually exclusive, please provide only one of {_all_doc_content_keys}'
```

Comment:
i think `content` and other keys are note mutually exclusive? e.g. we can have `content` and `uri` together

Reply:
uri is content now

Reply:
hmm not really? From my understand text/uri/blob/buffer are specific contents.
So if you set some string like 'hello world' as the content of a document it will have both text and context set to 'hello world'. Thus it does not really make sense to provide both content and text at the same time because it is in fact the same.

Reply:
you're right, ignore my comment.